CONTRIBUTING.md

```diff
@@ -36,6 +36,7 @@ To send us a pull request, please:
 4. Commit to your fork using clear commit messages.
 5. Send us a pull request, answering any default questions in the pull request interface.
 6. Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
+7. To spin up the platform tests, which test autogluon among macos and windows, comment on the PR with `/platform_tests`. It is recommended to run the platform tests only after you have passed the default CI.
```

Comment:
Will this only work if the commenter is an AutoGluon maintainer? We wouldn't want this to work if they are a first time contributor for example. We should clarify this.

Reply:
I think anyone commenting can spin up the test. It might be possible to do some check on the commenter, but I'll need to investigate a bit.

Reply:
By write permission do you mean only the core dev team? This is fine if its the case, we just want to avoid a situation where a random person could spam the command 100's of times.

Reply:
Basically whoever has the write to merge in the pull request.

Reply:
Ok, sounds good!